Directory.Packages.props

```diff
@@ -122,5 +122,12 @@
     <PackageVersion Include="xunit.extensibility.execution" Version="$(_XunitPackageVersion)" />
     <PackageVersion Include="xunit.runner.utility" Version="2.4.1" />
     <PackageVersion Include="Xunit.StaFact" Version="1.2.46-alpha" />
+
+    <!--
+      These package references exist solely to fix component governance issues. Referencing them here
+      at the appropriate version, in combination with using CentralPackageTransitivePinningEnabled, will
+      ensure the correct version is used in all places
+    -->
+    <PackageVersion Include="Microsoft.IO.Redist" Version="6.0.1" />
```

Comment:
Has the change I made in 17.8 flowed forward to this branch yet? If so, can we remove the extra packagerefs I needed to add?

Reply:
Dont' see it in main yet

Reply:
It's trying to come up in https://github.com/dotnet/razor/pull/10711

Reply:
Ah, I see https://github.com/dotnet/razor/pull/10711 hadn't made it because it was waiting for me for a ping I missed.

Reply:
That merged now so let me rebase on top of that.

Reply:
#10727 is merging that change into main. It had conflicts that @davidwengier fixed but that lead to build errors. I started to fix it up this morning and since I was already there I just merged in this change to that PR. Going to see how that goes. If it works I will close this one out.